Client/Frontend/Browser/BrowserViewController/BrowserViewController+ProductNotification.swift

```diff
@@ -68,6 +107,27 @@ extension BrowserViewController {
             notifyHttpsUpgrade(theme: Theme.of(selectedTab))
             notificationShown = true
         }
+        
+        // Step 5: Share Brave Benchmark Tiers
+        guard !notificationShown else { return }
+
+        if !Preferences.ProductNotificationBenchmarks.allTiersShown.value {
+            let numOfTrackerAds = BraveGlobalShieldStats.shared.adblock + BraveGlobalShieldStats.shared.trackingProtection
+            let existingTierList = BenchmarkTrackerCountTier.allCases.filter({ numOfTrackerAds > $0.rawValue })
+
+            for tier in existingTierList {
+                if Preferences.ProductNotificationBenchmarks.trackerTierCount.value < numOfTrackerAds {
```

Comment:
you could move this to `for-in` loop using `where` keyword, or convert to forEach.filter,
like `for tier in existingTierList where Preferences.ProductNotificationBenchmarks.trackerTierCount.value < numOfTrackerAds {`

removes indentation and is still readable, but it's up to you what you find clean to read

Reply:
I think I changed the logic inside and with it I forgot to add the If clause inside the where.